postcss.config.js

```diff
@@ -4,5 +4,15 @@ module.exports = {
   plugins: [
     require('autoprefixer'),
     require('postcss-import'),
+    require('@fullhuman/postcss-purgecss')({
```

Comment:
The styles doesn't get hot-reloaded unfortunately :[ (I have to restart the dev server if I want to include a new class name etc.)